cypress/integration/admin/config/authenticationSection.spec.js

```diff
@@ -0,0 +1,137 @@
+describe('Authentication Section', () => {
```

Comment:
Refactor (blocking): Instead of having this spec in the `admin` folder, rename it to `adminFlows` to keep it uniform.